docs/_macros/bndversion.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+class: Macro
+title: bndversion
+summary: Returns the current running bndversion
+---
+
+	Returns the currently running bnd version. E.g 5.3.0 
```

Comment:
You should also mention the returned version includes the micro. So it could be 5.3.1 for example.